src/benchmarklib/tpcc/procedures/tpcc_delivery.hpp

```diff
@@ -0,0 +1,22 @@
+#include "abstract_tpcc_procedure.hpp"
+
+namespace opossum {
+
+// Limitations:
+// 2.7.2   The delivery procedure is not executed in any type of deferred mode (given than we do not have the
+//         notion of terminals) and does not write a result file.
+// 2.7.4.2 We do not report a skipped delivery ratio over 1%
+
+class TPCCDelivery : public AbstractTPCCProcedure {
+ public:
+  TPCCDelivery(const int num_warehouses, BenchmarkSQLExecutor& sql_executor);
```

Comment:
I think we had that discussion a while back. While it only makes sense in the implementation, I like making the declaration consistent. In fact, I usually only type one of them and copy it, so that const ends up there automatically.